File: node/fizz-buzz.js
#!/usr/bin/node

const len = process.argv[2] || 100;

for( i = 1; i <= len; i++ ) {
	let fizz = i % 3 == 0 ? 'Fizz' : '';
	let buzz = i % 5 == 0 ? 'Buzz' : '';

	console.log(fizz + buzz ? fizz + buzz : i);
}

File: ruby/fizz-buzz.rb
#!/usr/bin/env ruby

len = (ARGV[0] || 100).to_i

len.times do |i|
  i += 1
  fizz = i % 3 == 0 ? 'Fizz' : ''
  buzz = i % 5 == 0 ? 'Buzz' : ''

  puts (fizz + buzz).length > 0 ? fizz + buzz : i
end
